internal/api/internalapi/client.go

```diff
@@ -33,9 +34,28 @@ func defaultFrontendInternal() string {
 type internalClient struct {
 	// URL is the root to the internal API frontend server.
 	URL string
+
+	grpcClient func() (proto.FrontendServiceClient, error)
 }
 
-var Client = &internalClient{URL: "http://" + frontendInternal}
+var Client = &internalClient{
+	URL: "http://" + frontendInternal,
+	// 	grpcClient: syncx.OnceValues(func() (proto.FrontendServiceClient, error) {
+	// 		serverURL := "http://" + frontendInternal
+	// 		u, err := url.Parse(serverURL)
+	// 		if err != nil {
+	// 			return nil, err
+	// 		}
+
+	// 		l := log.Scoped("frontendGRPCClient", "gRPC client for frontend internal api")
+	// 		conn, err := defaults.Dial(u.Host, l)
```

Comment:
```suggestion
	 		conn, err := defaults.Dial(u.Host, l)
```
I get a importcycle error when using internal/grpc/defaults as well .